web-services/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "web-services",
+  "version": "1.0.0",
+  "description": "server side of our trello kpi dashboard",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "sync": "cross-env aws s3 sync ../client s3://interns-oct2018 --delete --profile internship",
```

Comment:
As discussed, this should go into it's own package.json in the client directory.